libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -47,10 +48,9 @@
 </ng-container>
 <ng-template #empty>
   <div class="empty">
-    <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
-        >JavaScript</a
-      >".
+    <p class ="example-text">
+      Try searching for a topic, for example
+      <button (click)="searchExample()" aria-label ="JavaScript Example Button" style="cursor: pointer;">"JavaScript"</button>.
```

Comment:
Avoid using inline CSS

Reply:
Removed inline CSS and added styling in scss file